unlock-app/src/stories/creator/CreatorLock.stories.js

```diff
@@ -32,6 +42,7 @@ storiesOf('CreatorLock', CreatorLock)
   .addDecorator(getStory => <Provider store={store}>{getStory()}</Provider>)
   .add('Deployed', () => {
     const lock = {
+      id: '0xab',
```

Comment:
This is a a giant nit but the id is not the address and starting with `0x` implies an eth address? Would you be ok to use another random id?